COPY/etc/manageiq/postgresql.conf.d/01_miq_overrides.conf

```diff
@@ -11,7 +11,7 @@
 #------------------------------------------------------------------------------
 
 listen_addresses = '*'
-max_connections = 1000
+max_connections = 500
```

Comment:
It does, max_connections has a direct impact on `work_mem` tuning. This is where some dynamic tuning based on worker and process could would be beneficial. I will set this back to 1000

> This is very important to some of the below parameters (particularly work_mem) because there are some memory resources that are or can be allocated on a per-client basis, so the maximum number of clients suggests the maximum possible memory use. Generally, PostgreSQL on good hardware can support a few hundred connections.